controllers/login.js

```diff
@@ -0,0 +1,13 @@
+const bcrypt = require('bcrypt');
+const {getUserByEmail , users} = require('../database/users');
+
+const loginUser = (email,password) => {
+const user = getUserByEmail(email);
```

Comment:
very bad tabs in this function. please fix it because it hard to read

Reply:
fixed it